src/xcode/ENA/ENA/Source/Workers/Logging/Logging.swift

```diff
@@ -67,14 +67,16 @@ enum Log {
 	private static func log(message: String, type: OSLogType, log: OSLog, error: Error?, file: String, line: Int, function: String) {
 		#if !RELEASE
 		// Console logging
-		let meta: String = "[\(file):\(line)] [\(function)]"
-
-		// obviously we have to disable swiftline here:
-		// swiftlint:disable:next no_direct_oslog
-		os_log("%{private}@ %{private}@", log: log, type: type, meta, message)
-
-		// Save logs to File. This is used for viewing and exporting logs from debug menu.
-		fileLogger.log(message, logType: type, file: file, line: line, function: function)
+		DispatchQueue.global(qos: .background).async {
```

Comment:
Perhaps doing it on `.utility` and not `.background` to avoid being killed by the os for useless work too early?